src/Plugins/Collidable/README.md

```diff
@@ -5,24 +5,47 @@ the mirror movement for you. These currently only work with `Sortable`, `Swappab
 
 This plugin is not included by default, so make sure to import it before using.
 
-### Import
+### Usage
 
+- ES6:
 ```js
 import { Plugins } from '@shopify/draggable';
-```
-
-```js
+// Or
 import Collidable from '@shopify/draggable/lib/plugins/collidable';
```

Comment:
How about change those to 

```js
import { Draggable, Plugins } from "@shopify/draggable";
// Or
// import Draggable from '@shopify/draggable/lib/draggable';
// import Collidable from '@shopify/draggable/lib/plugins/collidable';

const draggable = new Draggable(document.querySelectorAll("ul"), {
  draggable: "li",
  collidables: ".other-list",
  plugins: [Plugins.Collidable], // Or [Collidable]
});
```

so that others can copy and paste it directly to use.